.buildkite/scripts/unit_tests.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+source .buildkite/scripts/install_tools.sh
+
+set -euo pipefail
+
+echo "--- Run Unit Tests"
+sudo chmod -R go-w "${BEATS_PROJECT_NAME}/"
+pushd "${BEATS_PROJECT_NAME}" > /dev/null
+
+umask 0022
```

Comment:
I cannot understand why those permissions are changed. Can you add a comment with the rationale and consequences if not done?

Reply:
added